region-cloner/scripts/gui.lua

```diff
@@ -39,7 +39,10 @@ gui.create_gui = function (player)
     clear_paste_area_table.add{type="label", caption="Clear paste area:", tooltip="Look for entities in the area about to be pasted into and delete them. If you don't clear normal entities it's likely that you'll stack mutiple on one tile, so be careful! Defaults to true."}
     clear_paste_area_table.add{type="checkbox", caption="Normal entities", state=true, name=GUI_ELEMENT_PREFIX .. "clear_normal_entities", tooltip="Clears the paste area of any player owned entities. If you use tile paste overrides with this setting enabled it probably won't work like you want it to!"}
     clear_paste_area_table.add{type="checkbox", caption="Resources", state=true, name=GUI_ELEMENT_PREFIX .. "clear_resource_entities", tooltip="Clears the paste area of ore/resources. May need to pre-generate the map to ensure ore is generated by the time we look for it."}
-
+	
+	--[Adding the "progress bar" checkbox]
+	advanced_settings_gui.add{type="checkbox", caption="Progress bar", state=false, name=GUI_ELEMENT_PREFIX .. "progress_bar", tooltip="The progress bar spends an additional 1 tick for each copy"}
+	advanced_settings_gui.add{type="checkbox", caption="Progress bar. Detailed log", state=false, name=GUI_ELEMENT_PREFIX .. "detailed_log", tooltip="The detailed log spends an additional 4 tick for each copy"}
```

Comment:
I'll clarify a bit on these tooltips that it increases the chance of clones breaking due to ticks advancing.